cmake/Spt3gIncludes.cmake.in

```diff
@@ -74,3 +74,39 @@ macro(add_spt3g_test test_name)
 		set_tests_properties(${PROJECT}/${test_name} PROPERTIES LABELS ${test_labels})
 	endif ()
 endmacro(add_spt3g_test test_name)
+
+macro(add_spt3g_test_program test_name)
+	cmake_parse_arguments("ADD_TEST_PROGRAM"
+	                      "" # options
+	                      "" # one value keywords
+	                      "SOURCE_FILES;TEST_LABELS;USE_PROJECTS" # multi-value arguments
+	                      ${ARGN}
+	                      )
+	
+	add_test(NAME ${PROJECT}/${test_name}
+	         WORKING_DIRECTORY ${SPT3G_BUILD_DIR}
+	         COMMAND ${PROJECT}-${test_name}
+	         )
+	
+	if(ADD_TEST_PROGRAM_TEST_LABELS)
```

Comment:
Something weird going on with the indentation in this code block.  Mixing tabs and spaces maybe?

Reply:
That was intentional, as the tabs provide the structural indentation, and the spaces align things within the lines, but I guess it's a dying style these days. I can make it use only one character is that is strongly preferred.

Reply:
I guess the github web interface just formats it oddly?  Some of the lines don't look like they're aligned like you'd expect.

Reply:
Curious; it seems to work as expected for me even when I adjust the displayed tab width:
4 space tab stops:
<img width="353" alt="Screen Shot 2023-04-07 at 15 36 49" src="https://user-images.githubusercontent.com/5001008/230667353-7ad8f684-f51b-4d47-8513-158818228724.png">
8 space tab stops:
<img width="382" alt="Screen Shot 2023-04-07 at 15 36 19" src="https://user-images.githubusercontent.com/5001008/230667401-b5fe0cf0-b1fc-4630-9321-cf96e7146843.png">

Reply:
Oh!  It's the blocks below that, lines 91-99.

Reply:
Thanks, there was a mistake there, which is hopefully now fixed.